Remotion/Web/Core/ExecutionEngine/UrlMapping/UrlMappingConfiguration.cs

```diff
@@ -56,11 +55,11 @@ public static UrlMappingConfiguration Current
 
   private static UrlMappingConfiguration CreateConfig ()
   {
-    string mappingFile = WebConfiguration.Current.ExecutionEngine.UrlMappingFile;
+    var mappingFile = SafeServiceLocator.Current.GetInstance<WxeUrlSettings>().UrlMappingFile;
     if (string.IsNullOrEmpty(mappingFile))
```

Comment:
Yes, when using create with an empty string as a UrlMappingFile

Reply:
hmm... okay, please switch to null instead of empty (feels cleaner in today's age) and we further clean this up when we also update the UrlMappingConfig reading.